policies/vla_eval.py

```diff
@@ -95,50 +101,73 @@ def main(_):
 
     # format the prompt
     prompt = f"In: What action should the robot take to {FLAGS.text_cond}?\nOut:"
+    print(prompt)
+    def run_continuous_img_primary(manipulator_interface):
```

Comment:
Can we move this into the `ActionClientInterface()` impl? so this bug fix can be applied to others that are using the `ActionClientInterface` api